test/test_browser.py

```diff
@@ -2108,7 +2106,7 @@ def test_cubegeom_pre3(self):
   def test_cubegeom(self, args):
     # proxy only in the simple, normal case (we can't trace GL calls when
     # proxied)
-    self.btest('third_party/cubegeom/cubegeom.c', reference='third_party/cubegeom/cubegeom.png', args=['-O2', '-g', '-sLEGACY_GL_EMULATION', '-lGL', '-lSDL'] + args, also_proxied=not args)
```

Comment:
Why remove the extra coverage here?

Reply:
Ah.. I think that might have been left over from testing.  Reverted.